plugin/src/main/java/hudson/plugins/swarm/KeepSwarmClientNodeProperty.java

```diff
@@ -6,6 +6,7 @@
 import hudson.slaves.NodePropertyDescriptor;
 import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.springframework.lang.NonNull;
```

Comment:
Wrong annotation. Please use `edu.umd.cs.findbugs.annotations.NonNull` for consistency with Jenkins core.

Reply:
:man_facepalming: Fixed, thanks.